Modules/Award.lua

```diff
@@ -234,6 +234,42 @@ local function AwardItem(player, cost, boss, zone, loot, reassign)
 			SendChatMessage(L["CONGRATS"].." "..winner.." "..L["ON"].." "..loot.." @ "..-cost.." "..L["DKP"], "GUILD")
 		end
 
+		if core.DB.defaults.AutoAward then

```

Comment:
this should be core.DB.defaults.AutoAwardLoot

Reply:
Good catch. Fixed.